package.json

```diff
@@ -45,5 +45,13 @@
     "stylelint": "^13.9.0",
     "stylelint-config-recommended": "^3.0.0",
     "stylelint-config-styled-components": "^0.1.1"
+  },
+  "dependencies": {
+    "@polkadot/api": "^4.0.3",
+    "bs58": "^4.0.1",
```

Comment:
This we should not import directly - the util wraps this.

Reply:
Okay, @AmazingBeerBelly let's remove this dep and use functions in util-crypto